tests/security/audit/aureport.pm

```diff
@@ -0,0 +1,55 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-Later
+#
+# Summary: Verify the "aureport" utility can generate default audit report and reports with specifc parameters
+# Maintainer: llzhao <llzhao@suse.com>, shawnhao <weixuan.hao@suse.com>
+# Tags: poo#81772
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my $audit_log = '/var/log/audit/audit.log';
+
+    select_console 'root-console';
+
+    # Make sure audit service is started
+    assert_script_run('systemctl is-active auditd');
+
+    # Generate audit records for testing
+    assert_script_run("echo '' > $audit_log");
+    assert_script_run('systemctl stop apparmor');
+    assert_script_run('systemctl start apparmor');
+
+    # Stop auditd before checking records in case the following executions will generate new audit records
+    assert_script_run('systemctl stop auditd');
+
+    # Generate report by default
+    assert_script_run('aureport');
+
+    # Generate AVC report
+    validate_script_output('aureport -a', sub { m/AVC/ });
```

Comment:
It is optional:
Could you please compare the values of $num1 and $num2 to check if they are the same?
$numb1 is the "Number of AVC's: $number" in the outputs of "assert_script_run('aureport');"
$numb2 is the total number for the records in 'aureport -a'
See https://bugzilla.suse.com/tr_show_case.cgi?case_id=1768552 for the outputs examples.

Reply:
`aureport - a tool that produces summary reports of audit daemon logs` so adding this test step can verify the summary is correct.

Reply:
Sure, I will add the comparison

Reply:
AVC report counts checking added now.